cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -378,6 +378,11 @@ runTxBuildRaw (AnyCardanoEra era)
     firstExceptT ShelleyTxCmdWriteFileError . newExceptT $
       writeFileTextEnvelope fpath Nothing txBody
 
+queryErrorToShelleyTxCmdError :: QueryError ShelleyTxCmdError -> ShelleyTxCmdError
```

Comment:
Likewise here, why not add a constructor to `ShelleyTxCmdError`? E.g `ShelleyTxQueryRelatedCmdError QueryError`?